src/pages/matching/survey/index.tsx

```diff
@@ -0,0 +1,221 @@
+            </FeedbackTypeCheckbox>
+            <FeedbackTypeCheckbox
+              checked={feedbackTypeObj.isBad}
+              onChange={(e) => {
+                onChangeCheckValue('isBad', e.target.checked);
+              }}
+            >
+              <SurveyBadIcon style={feedbackTypeObj.isBad ? activeStyle : {}} />
+              <RegularSpan>불쾌해요</RegularSpan>
+            </FeedbackTypeCheckbox>
+          </Grid2Column>
+          <div>
+            <RegularParagraph07>
+              상대방에대한 피드백을 작성해주세요.
+              <br />
+              불쾌해요를 선택하셨다면 이유를 남겨주세요.
+            </RegularParagraph07>
+            <TextArea
+              placeholder="피드백을 남겨주세요."
+              value={feedbackMsg}
+              onChange={(e) => {
+                setFeedbackMsg(e.target.value);
+              }}
+            />
+          </div>
+          <CheckBoxLabel>
+            <input
+              type="checkbox"
+              checked={isBan}
+              onChange={(e) => {
+                setIsBan(e.target.checked);
+              }}
+            />
+            <RegularSpan>이 사람이랑 전화하지 않을래요.</RegularSpan>
+          </CheckBoxLabel>
+          <ButtonStyle
+            type="submit"
+            isActive={feedbackMsg !== ''}
+            disabled={feedbackMsg === ''}
+          >
+            피드백 작성 완료
+          </ButtonStyle>
+        </form>
+      </Section>
+    </>
+  );
+};
+
+export default MatchingSurvey;
+
+const Section = styled.section`
+  text-align: center;
+  margin-top: 54px;
+  padding: 0 20px;
+`;
+
+const Title = styled.h1`
+  ${ScreenReaderOnly}
+`;
+
+const Grid2Column = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  justify-content: space-evenly;
+  row-gap: 30px;
+  margin-bottom: 60px;
+  label {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 6px;
+  }
+`;
+
+const BoldParagraph = styled.p`
+  ${({ theme }) => theme.fonts.headline_02};
+  margin-bottom: 10px;
+`;
+
+const RegularParagraph04 = styled.p`
+  ${({ theme }) => theme.fonts.body_04};
+  color: ${({ theme }) => theme.colors.gray_02};
+  margin-bottom: 40px;
+`;
+
+const RegularParagraph07 = styled.p`
+  ${({ theme }) => theme.fonts.body_07};
+  color: ${({ theme }) => theme.colors.gray_00};
+`;
+
+const RegularSpan = styled.span`
+  ${({ theme }) => theme.fonts.body_04};
+  color: ${({ theme }) => theme.colors.gray_00};
+`;
+
+const TextArea = styled.textarea`
+  ${({ theme }) => theme.fonts.body_07};
+  color: ${({ theme }) => theme.colors.gray_00};
+  border: 1px solid ${({ theme }) => theme.colors.gray_06};
+  border-radius: 16px;
+  width: 100%;
+  min-height: 100px;
+  overflow: auto;
+  padding: 16px;
+  margin-top: 10px;
+`;
+
+const CheckBoxLabel = styled.label`
+  float: left;
+  margin-top: 40px;
+  input {
+    margin-right: 10px;
+  }
+`;
+
+const ButtonStyle = styled.button<{
+  isActive: boolean;
+}>`
+  ${theme.fonts.button_02};
```

Comment:
p1;
ThemeProvider 를 통해 전달된 theme 객체 사용과 styles 파일을 직접 import 하여 사용하는 theme 객체 사용을 혼용하고 있습니다! ThemeProvider를 통한 스타일로 통일 시켜주세요~
```suggestion
  ${({ theme }) => theme.fonts.button_02};
```

Reply:
넵! 수정하였습니다